Sources/VSM/Documentation.docc/VSM.md

```diff
@@ -8,7 +8,7 @@ VSM is a reactive, unidirectional, type-safe, behavior-driven, clean architectur
 
 ![VSM Overview Diagram](vsm-diagram.png)
 
-This package provides helpful types for implementing VSM, such as the ``StateContainer`` type which manages the current `State`. `UIViews`, `UIViewControllers`, or SwiftUI `Views` that conform to ``ViewStateRendering`` can easily react to changes in `State` by rendering the current `State`.
+This package provides helpful types for implementing VSM, such as the ``ViewState`` type (or the ``RenderedViewState`` type for UIKit views) which manages and renders the current `State`.
```

Comment:
While `ViewState & `RenderedViewState` are types... aren't they referred to more often as property wrappers? IDK, it's a little weird. Maybe I'm over thinking it...